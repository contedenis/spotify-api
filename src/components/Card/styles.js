// @packages
import styled from 'styled-components';

const CardStyled = styled.div`
  border-radius: 5px;
  background: #7FFFD4;
`;

export {
  CardStyled,
};
