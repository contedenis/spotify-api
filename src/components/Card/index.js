// @packages
import React from 'react';
import PropTypes from 'prop-types';

// @own
import { CardStyled } from './styles';

function Image({ children }) {
  return (
    <CardStyled>
      {children}
    </CardStyled>
  );
}

Image.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Image;
