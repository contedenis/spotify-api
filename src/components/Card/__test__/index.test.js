// @packages
import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

// @own
import Card from '../index';

describe('Card', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <Card />,
    );

    expect(container.firstChild).toMatchSnapshot();
  });

  it('render Card', () => {
    render(<Card>Item</Card>);

    const cardItem = screen.getByText('Item');
    expect(cardItem).toBeInTheDocument();
  });
});
