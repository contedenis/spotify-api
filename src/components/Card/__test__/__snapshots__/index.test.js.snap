// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches snapshot 1`] = `
<div
  class="sc-bdnylx aHbng"
/>
`;
